docs/GITOPS-CONFIG.md

```diff
@@ -20,31 +20,43 @@ If you installed using the [`ai-rhdh-installer`](../README.md#install) all that
 
 #### Step 2: Updating Plugins
 **Note: RHDH will encounter errors if the ArgoCD plugins are loaded before an instance is properly attached - This step may result in errors until all steps are completed**
 
-To include the ArgoCD plugins we need to edit the dynamic plugins ConfigMap that was created by the RHDH Operator:
+To include the [ArgoCD plugins list](../dynamic-plugins/argocd-plugins.yaml) we need to edit the dynamic plugins ConfigMap that was created by the RHDH Operator:
 
 ![Dynamic Plugins Example](../assets/dynamic-plugins-example.png)
 
-Edit the associated `yaml` file to include `argocd-plugins.yaml` under the `includes` section:
+Edit the associated `yaml` file to include the contents of the [ArgoCD plugins list](../dynamic-plugins/argocd-plugins.yaml) under the `plugins` section:
 
 ![Dynamic Plugins Example 2](../assets/dynamic-plugins-example-2.png)
 
-#### Step 3: Updating RHDH Deployment
-Now that all of the required ConfigMaps and Secrets are apart of the namespace, as well as adding the plugins we must edit the RHDH Deployment to properly reference these items.
-
-Since we are referencing `argocd-plugins.yaml` in our dynamic-plugins ConfigMap we first must add `argocd-plugins.yaml` as a `volumeMount` to the `initContainers` field in the RHDH Deployment under `.spec.template.spec.initContainers.volumeMounts`
+Alternatively, we can use this series of commands to perform the same task with `kubectl` and `yq` using the [`argocd-plugins.yaml`](../dynamic-plugins/argocd-plugins.yaml):
+
+1. Fetch the dynamic plugins ConfigMap and save the `dynamic-plugins.yaml` content within to a temp file
+    ```sh
+    kubectl get configmap backstage-dynamic-plugins-ai-rh-developer-hub -n ai-rhdh -o yaml | yq '.data["dynamic-plugins.yaml"]' > temp-dynamic-plugins.yaml
+    ```
+2. Merge the contents of [`argocd-plugins.yaml`](../dynamic-plugins/argocd-plugins.yaml) into the temp file
+    ```sh
+    yq -i ".plugins += $(yq '.plugins' ./dynamic-plugins/argocd-plugins.yaml -M -o json) | .plugins |= unique_by(.package)" temp-dynamic-plugins.yaml
+    ```
+3. Patch the dynamic plugins ConfigMap with the updated content in the temp file
+    ```sh
+    kubectl patch configmap backstage-dynamic-plugins-ai-rh-developer-hub -n ai-rhdh \
+    --type='merge' \
+    -p="{\"data\":{\"dynamic-plugins.yaml\":\"$(echo "$(cat temp-dynamic-plugins.yaml)" | sed 's/"/\\"/g' | sed 's/$/\\n/g' | tr -d '\n')\"}}"
+    ```
+4. Dynamic plugins should be updated with the [ArgoCD plugins list](../dynamic-plugins/argocd-plugins.yaml) with a pod update triggered and you may remove the temp file at this point
 
-![ArgoCD Plugins Addition](../assets/argocd-plugins-example.png)
+#### Step 3: Updating RHDH Deployment
+Now that all of the required ConfigMaps and Secrets are apart of the namespace, we must edit the RHDH Deployment to properly reference these items.
```

Comment:
Might be my english here, just not quite sure if "are apart of" == "are part of" 
```suggestion
Now that all of the required ConfigMaps and Secrets are part of the namespace, we must edit the RHDH Deployment to properly reference these items.
```